commandapi-annotations-reloaded-tests/src/test/resources/DocumentationWarpCommand.java

```diff
@@ -0,0 +1,155 @@
+/*******************************************************************************
+ * Copyright 2018, 2021 Jorel Ali (Skepter) - MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *******************************************************************************/
+import java.util.HashMap;
+import java.util.Map;
+
+import org.bukkit.Location;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.java.JavaPlugin;
+
+import dev.jorel.commandapi.CommandAPI;
+import dev.jorel.commandapi.CommandAPICommand;
+import dev.jorel.commandapi.annotations.reloaded.annotations.Command;
+import dev.jorel.commandapi.annotations.reloaded.annotations.Permission;
+import dev.jorel.commandapi.annotations.reloaded.annotations.Subcommand;
+import dev.jorel.commandapi.annotations.reloaded.arguments.AStringArgument;
+import dev.jorel.commandapi.arguments.ArgumentSuggestions;
+import dev.jorel.commandapi.arguments.StringArgument;
+
+/* ANCHOR: warps */
+/* ANCHOR: warps_command */
+@Command("warp")    
+public class DocumentationWarpCommand {
```

Comment:
In general, the anchor names should be the name of the documentation page that they appear in. The first example would be something like `page_name1` or `pageName1` and so on.

Reply:
I have no clue how the documentation pages work, however everything will need to be properly documented before `dev/annotations-reloaded` can be merged into `dev/dev` for 10.0. We should probably create an issue for doing this, it will need to be the last thing we do once the full annotation spec is finalised. 

For this PR, `dev/annotations-reloaded` <- `dev/annotations-reloaded`, I think we should focus on making sure this is the base we want to build upon for the new annotations system. We can then create all the issues we need for missing features and improvements, and go at it piecemeal from there.